### PR TITLE
Refactored functions to determine current day

### PR DESCRIPTION
**Pull request by ngwatso**
# Description

This PR was created to refactor the functions to determine current day in the CurrentActivity and NextActivity components.  Previous functions were large and not easily readable; refactoring brought these functions down from 61 lines to 11 lines, while maintaining functionality.  One small change was also done in the CountdownTimer component, added NextActivty component to be rendered in the heads up display.

## Commits Checklist
- [x] commits have clear title names 
- [x] commit has descriptive message of what has changed 
- [x] more small commits than less big commits 

## Code Checklist 
- [x] code is formatted appropriately 
- [x] code meets DRY standards
- [x] no dead code (check for logs and print statements)  
- [x] no TODOS instead necessary comments 
- [x] code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries

### CODE REVIEW

File: src/components/pages/HeadsUpDisplay/CurrentActivity.js [hunk 1]
```diff
@@ -1,72 +1,35 @@
...
         return;
-      } else if (day === 12) {
-        setDay(13);
+      } else if (day === prevDay[0]) {
+        prevDay[0] === 13 ? setDay(0) : setDay(prevDay[0] + 1);
         return;
       }
-    }
+    };
+    calcDay();
```
Comment (Jtamedrano) [inline src/components/pages/HeadsUpDisplay/CurrentActivity.js]: This was a very clean optimization

File: src/components/pages/HeadsUpDisplay/NextActivity.js [hunk 2]
```diff
@@ -88,14 +50,15 @@ export const NextActivity = () => {
...
-    setActivity(activityList[day+1]);
+    setActivity(activityList[day + 1]);
     return activity;
   };
 
+  // useEffect to call both functions on render
   useEffect(() => {
     nextDay();
     nextActivity();
   });
```
Comment (Jtamedrano) [inline src/components/pages/HeadsUpDisplay/NextActivity.js:33]: I still think there should be some sort of dependency with eslint ignore here. wouldn't want this to an infinite loop.

**Review by Jtamedrano — APPROVED** [review]
Very Nice Work!

**Review by ashton-stom — COMMENTED** [review]
This is much better. Definitely easy to read and more organized.

File: src/components/pages/HeadsUpDisplay/NextActivity.js (hunk 2 above)
Comment (ngwatso) [reply src/components/pages/HeadsUpDisplay/NextActivity.js:33]: Thanks for the input Jesse!  I have tried using every variation of dependancies that I could think of, some resulted in infinite loops and others caused the component to not re-render.  I am open to suggestions if you have any, but this is the only set up that I could find functions properly.  I did run through the full cycle many times with no infinite loop, but if you see something I'm missing I will try it out.

**Review by Jtamedrano — CHANGES_REQUESTED** [review]
Looks great over all, just a couple simple changes should cover it.

File: src/components/pages/HeadsUpDisplay/CurrentActivity.js [hunk 3]
```diff
@@ -41,14 +19,7 @@ export const CurrentActivity = () => {
...
       5: "Point Share",
-      6: "Matchup 1",
-      7: "Voting Session 1",
-      8: "Matchup 2",
-      9: "Voting Session 2",
-      10: "Matchup 3",
-      11: "Voting Session 3",
-      12: "Matchup 4",
-      13: "Big Final Reveal"
+      6: "Big Final Reveal"
```
[the remark this answers is not in the document]
Comment (ngwatso) [reply src/components/pages/HeadsUpDisplay/CurrentActivity.js]: Yeah, I missed that, I'll get the wording changed.

[the remark this answers is not in the document]
Comment (ngwatso) [reply src/components/pages/HeadsUpDisplay/CurrentActivity.js]: Wording has been changed on these.

File: src/components/pages/HeadsUpDisplay/CurrentActivity.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ export const CurrentActivity = () => {
       3: "Write",
       4: "Squadding Up",
       5: "Point Share",
-      6: "Big Final Reveal"
+      6: "Voting"
```
Comment (Jtamedrano) [inline src/components/pages/HeadsUpDisplay/CurrentActivity.js:22]: Thanks for getting this changed over

File: src/components/pages/HeadsUpDisplay/NextActivity.js [hunk 5]
```diff
@@ -20,9 +20,9 @@ export const NextActivity = () => {
       3: "Write",
       4: "Squadding Up",
       5: "Point Share",
-      6: "Big Final Reveal"
+      6: "Voting"
```
Comment (Jtamedrano) [inline src/components/pages/HeadsUpDisplay/NextActivity.js:23]: Thanks for getting this changed over

File: src/components/pages/HeadsUpDisplay/NextActivity.js [hunk 6]
```diff
@@ -20,9 +20,9 @@ export const NextActivity = () => {
...
     };
-    setActivity(day === 6 ? activityList[1] : activityList[day + 1]);
+    setActivity(day === 6 ? activityList[0] : activityList[day + 1]);
```
Comment (Jtamedrano) [inline src/components/pages/HeadsUpDisplay/NextActivity.js:25]: nice catch

**Review by Jtamedrano — APPROVED** [review]

**Review by ashton-stom — APPROVED** [review]
This will work as intended now. Nice catch on the activities.

**Pull request merged**